packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/FIAPPaymentQueueDeleteTests.m

```diff
@@ -85,6 +86,7 @@ - (void)testShouldContinueTransaction_should_default_to_yes {
 }
 
 - (void)testShouldShowPriceConsentIfNeeded {
+#if TARGET_OS_IOS
```

Comment:
Nit: Let's move this outside the test rather than inside, so the test doesn't show at all rather than appearing in UI as a passing test.